src/index.js

```diff
@@ -1,11 +1,39 @@
-const { app, BrowserWindow } = require('electron');
+const { app, BrowserWindow, ipcMain } = require('electron');
+const Datastore = require('@seald-io/nedb');
 const path = require('path');
 
+const userConfigDb = new Datastore({ filename: path.join(process.cwd(), './config/user-config.db'), autoload: true });
+
 // Handle creating/removing shortcuts on Windows when installing/uninstalling.
 if (require('electron-squirrel-startup')) { // eslint-disable-line global-require
   app.quit();
 }
 
+ipcMain.handle('save-user-config', async (event, arg) => {
+  if (typeof arg === 'object' && arg !== null && arg.hasOwnProperty('key') && arg.hasOwnProperty('value')) {
+    const { key, value } = arg
+    userConfigDb.update({ key }, {
+      key,
+      value
+    }, { upsert: true }, (err, numReplaced, upsert) => {
+      // TODO: handle errors
+    })
+  }
+})
+
+ipcMain.handle('load-user-config', async (event, arg) => {
+  if (typeof arg === 'string') {
+    return new Promise((resolve, reject) => {
+      userConfigDb.findOne({ key: arg }, (err, { value }) => {
```

Comment:
The error is on this line.

Reply:
Maybe it's because .config/user-config.db doesn't exist and doesn't contain the required keys?